packages/react-components/src/components/Loader/Loader.tsx

```diff
@@ -9,11 +9,21 @@ const baseClass = 'loader';
 const spinnerClass = `${baseClass}__spinner`;
 
 export interface LoaderProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Specify the loader size
+   */
   size?: 'small' | 'medium' | 'large';
+  /**
+   * Specify the loader label
+   */
   label?: string;
-  /** Fragment circle color */
+  /**
+   * Fragment circle color
+   */
   primaryColor?: string;
-  /** Main circle color */
+  /**
+   * Main circle color
```

Comment:
'Main' or 'Secondary'? Can you please rephrase the hint text for `primaryColor` and `secondaryColor` props?